src/index.ts

```diff
@@ -1,6 +1,6 @@
 import Ajv from "ajv";
 import { compare, genSalt, hash } from "bcryptjs";
-import express from "express";
+import express, { Request, Response } from "express";
```

Comment:
las importaciones de ```Request``` y ```Response``` son innecesarias, tienes que regresar este archivo a su estado original, para esto intenta hacer un ```git checkout master src/index.ts```.
```suggestion
import express from "express";
```